cmd/zoekt-webserver/main.go

```diff
@@ -504,3 +510,70 @@ var (
 		Help: "The total number of errors from zoekt watchdog.",
 	})
 )
+
+func mustRegisterMemoryMapMetrics(l sglog.Logger) {
+	if runtime.GOOS != "linux" {
+		l.Debug(
+			"skipping registration: must be running on a linux-based operating system",
+			sglog.String("current_operating_system", runtime.GOOS),
+			sglog.String("desired_operating_system", "linux"))
+
+		return
+	}
```

Comment:
this seems spammy for dev? Only place it would run. Does it feel safe to just not log in this condition, it is just a metric after all.